apps/web/pages/api/book/event.ts

```diff
@@ -362,6 +382,17 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
       await verifyAccount(web3Details.userSignature, web3Details.userWallet);
     }
 
+    const eventTypeRel = !eventTypeId
+      ? {}
+      : {
+          connect: {
+            id: eventTypeId,
+          },
+        };
+
+    const dynamicEventSlugRef = !eventTypeId ? eventTypeSlug : null;
+    const dynamicGroupSlugRef = !eventTypeId ? (reqBody.user as string).toLowerCase() : null;
```

Comment:
We need `dynamicEventSlugRef` and `dynamicGroupSlugRef` to store in the booking so that we can then use these later in case of rescheduling when we're dealing with dynamic group booking since we don't have an eventTypeId or a Team ID to identify these dynamic groups and events later on for reference.